transform/instance.go

```diff
@@ -107,13 +111,58 @@ func (at *arrayTransformer) baseValue(in interface{}, path string, modifier path
 	return nil, false, nil
 }
 
+func (at *arrayTransformer) baseValueXML(in interface{}, path string, modifier pathModifier) ([]interface{}, bool, error) {
+	// 1. Use a transform if it exists
+	if at.transforms != nil {
+		rawValue, err := at.transforms.transform(in, "array", modifier, at.format)
+		if err != nil {
+			return nil, false, err
+		}
+
+		//if rawValue is an array of xml nodes we need to append them to newValue for return as []interface{}
+		xmlNodeArray, ok := rawValue.([]*xmlquery.Node)
```

Comment:
is this block to just change the underlying type of the interface to `xmlquery.Node`s?